### ISSUE TITLE
Compilation error

### ISSUE THREAD

**Issue by BenoitEirik** [psaraci/risipapp#8]
Hello,

I try to compile your risiapp on Android but I have this error:
`clang++: error: no such file or directory: '@armeabi-v7a\object_script...\risipapp\bin\libRisipApp_armeabi-v7a.so.Armeabi-v7a'`
There is nothing in bin folder.

Do you know what I can do ?

**Comment by kastravec** [comment]
Yes, as the error says the risip library is missing, it is not present in the ...\risipapp\bin\ folder as it should be. Make sure you have it compiled and present in that folder.

**Comment by BenoitEirik** [comment]
All right, thank you. Could you tell me how I should compile the library for Android?

**Comment by kastravec** [comment]
Did you follow the steps as described in the README file? If follow you the steps, then it should and will build automatically for you..

copied from the README

- Clone the risipapp git repo.
- Inside the risipapp folder, clone the risip library (a.k.a. risip-voipsdk repo ). The risip repo 
https://github.com/psaraci/risip
- Build and run RisipApp directly in Qt Creator.

Did you do as above?

**Comment by BenoitEirik** [comment]
Yes, of course I followed the README. But that's precisely where I have this problem.

**Comment by kastravec** [comment]
Maybe something went wrong with compiling the risip library itself and therefore it is not copied automatically. 
Can you send more messages from the compiler output? Can you check if actually risip library is built at all?

**Comment by kastravec** [comment]
In your initial comment i can see that it is a clang compiler that you are using, but the Pjsip library binaries that i package together with risip are built with gcc. This means that you need to rebuild tje pjsip library yourself using clang, include the libraries in the respective risip folder and then give another risipapp a try, that should work

**Comment by BenoitEirik** [comment]
I don't have gcc for Android. I only have Clang, who was offered to me in "Kits". I don't know why. I'll try what you told me, compile PJSIP with Clang.

**Comment by kastravec** [comment]
Give it a try, let me know how it goes.

**Comment by BenoitEirik** [comment]
I can't compile PJSIP for Android on Windows because configure-android is only executable on Linux and I don't work on it at the moment.
Else I want to compile risipapp for windows but don't find pjsua2.hpp:
`In file included from ..\risipapp\risip\src\risipsdk\headers/risipaccount.h:25:0,
                 from ..\risipapp\risip\src\risipsdk\risipaccount.cpp:21:
..\risipapp\risip\src\risipsdk\headers/risipaccountconfiguration.h:24:10: fatal error: pjsua2.hpp: No such file or directory
 #include <pjsua2.hpp>
          ^~~~~~~~~~~~
compilation terminated.
In file included from ..\risipapp\risip\src\risipsdk\headers/risipaccount.h:25:0,
                 from ..\risipapp\risip\src\risipsdk\headers/risipbuddy.h:25,
                 from ..\risipapp\risip\src\risipsdk\risipbuddy.cpp:21:
..\risipapp\risip\src\risipsdk\headers/risipaccountconfiguration.h:24:10: fatal error: pjsua2.hpp: No such file or directory
 #include <pjsua2.hpp>
          ^~~~~~~~~~~~
compilation terminated.
mingw32-make: *** [Makefile:3422: risipaccount.o] Error 1
mingw32-make: *** Waiting for unfinished jobs....
mingw32-make: *** [Makefile:3630: risipbuddy.o] Error 1
In file included from ..\risipapp\risip\src\risipsdk\risipaccountconfiguration.cpp:20:0:
..\risipapp\risip\src\risipsdk\headers/risipaccountconfiguration.h:24:10: fatal error: pjsua2.hpp: No such file or directory
 #include <pjsua2.hpp>
          ^~~~~~~~~~~~
compilation terminated.
mingw32-make: *** [Makefile:3932: risipaccountconfiguration.o] Error 1
In file included from ..\risipapp\risip\src\risipsdk\headers/risip.h:25:0,
                 from ..\risipapp\risip\src\risipsdk\risip.cpp:21:
..\risipapp\risip\src\risipsdk\headers/risipendpoint.h:26:10: fatal error: pjsua2.hpp: No such file or directory
 #include <pjsua2.hpp>
          ^~~~~~~~~~~~
compilation terminated.
mingw32-make: *** [Makefile:3325: risip.o] Error 1
In file included from ..\risipapp\risip\src\risipsdk\headers/risipcall.h:24:0,
                 from ..\risipapp\risip\src\risipsdk\risipcall.cpp:21:
..\risipapp\risip\src\risipsdk\headers/risipmedia.h:26:10: fatal error: pjsua2.hpp: No such file or directory
 #include <pjsua2.hpp>
          ^~~~~~~~~~~~
compilation terminated.
mingw32-make: *** [Makefile:3541: risipcall.o] Error 1
In file included from ..\risipapp\risip\src\risipsdk\risipmedia.cpp:21:0:
..\risipapp\risip\src\risipsdk\headers/risipmedia.h:26:10: fatal error: pjsua2.hpp: No such file or directory
 #include <pjsua2.hpp>
          ^~~~~~~~~~~~
compilation terminated.
In file included from ..\risipapp\risip\src\risipsdk\risipendpoint.cpp:21:0:
..\risipapp\risip\src\risipsdk\headers/risipendpoint.h:26:10: fatal error: pjsua2.hpp: No such file or directory
 #include <pjsua2.hpp>
          ^~~~~~~~~~~~
compilation terminated.
mingw32-make: *** [Makefile:3798: risipendpoint.o] Error 1
mingw32-make: *** [Makefile:3716: risipmedia.o] Error 1
In file included from ..\risipapp\risip\src\risipsdk\risipmessage.cpp:20:0:
..\risipapp\risip\src\risipsdk\headers/risipmessage.h:25:10: fatal error: pjsua2.hpp: No such file or directory
 #include <pjsua2.hpp>
          ^~~~~~~~~~~~
compilation terminated.
mingw32-make: *** [Makefile:3855: risipmessage.o] Error 1
16:31:11: Le processus "D:\Qt\Tools\mingw730_64\bin\mingw32-make.exe" s'est terminé avec le code 2.
Erreur lors de la compilation/déploiement du projet risipapp (kit : Desktop Qt 5.14.1 MinGW 64-bit)
When executing step "Make"
16:31:11: Elapsed time: 00:04.`

**Comment by kastravec** [comment]
That's because I don't ship any PJSIP binaries nor any pjsip include files with the risip library, you need to do that yourself i'm afraid.

https://github.com/psaraci/risip/tree/master/platforms

the win folder is empty
